redis/src/commands/mod.rs

```diff
@@ -88,10 +88,16 @@ implement_commands! {
     }
 
     /// Sets multiple keys to their values.
+    #[deprecated(since = "0.24.0", note = "Renamed to mset() to reflect Redis name")]
```

Comment:
Sorry, one more nit: the initial deprecated release is likely to be 0.22.4, not 0.24.0.